config/test.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```

Comment:
During the session I forgot to mention that we also want to add a `PORT` property on the test environment config object so that our tests on Travis work whenever we configure them.